src/gui/base/BubbleTextField.js

```diff
@@ -337,30 +360,30 @@ export class BubbleTextField<T> {
 	}
 
 	handleUpArrow(): boolean {
-		if (this.selectedSuggestion != null) {
-			this.selectedSuggestion.selected = false
-			let next = (this.suggestions.indexOf(this.selectedSuggestion) - 1) % this.suggestions.length
+		if (this.selectedSuggestion() != null) {
+			neverNull(this.selectedSuggestion()).selected = false
+			let next = (this.suggestions.indexOf(this.selectedSuggestion()) - 1) % this.suggestions.length
 			if (next === -1) {
 				next = this.suggestions.length - 1
 			}
-			this.selectedSuggestion = this.suggestions[next]
-			this.selectedSuggestion.selected = true
+			this.suggestions[next].selected = true
+			this.selectedSuggestion(this.suggestions[next])
 		}
 		return false
 	}
 
 	handleDownArrow(): boolean {
-		if (this.selectedSuggestion != null) {
-			this.selectedSuggestion.selected = false
-			let next = (this.suggestions.indexOf(this.selectedSuggestion) + 1)
+		if (this.selectedSuggestion() != null) {
+			neverNull(this.selectedSuggestion()).selected = false
```

Comment:
I would personally rather put `this.selectedSuggestion()` in a variable first over using `neverNull`, here as well as further down

Reply:
Yeah, it's probably nicer. Amended and pushed the change.